### DISCUSSION TITLE
Is the config stored on the dongle or on a file?

### DISCUSSION THREAD

**Discussion by gcannar** [Koenkk/zigbee2mqtt#6487] [Devices]
Hi there,

I' m using zigbee2mqtt with Domoticz but wanna give a try to HA on another sd card. The question is, installin and configuring all the device on the new OS will brick the config? 
After try HA can i put back my old SD with domotcz and all will be back safe?
I mean...the pairing and all the config are stored on the dongle itself so install and pairing new devices under a new OS will cause problem reverting back?

Thanks, G.

**Comment by m0wlheld** [comment]
Zigbee2mqtt is connected to MQTT only, so you install HA with MQTT integration, enable HA MQTT discovery support in Z2M and devices will be there. 

You only need to re-pair, if you use an integration that is accessing coordinator hardware directly, like ZHA or deCONZ.

**Comment by gcannar** [comment]
Ye i know! But on the new sd i need a new inatallation of zigbee2mqtt.
So...i have to copy the database fron the orher sd or i just installe z2m
and the device will ne there on the coordinator?

Il dom 28 feb 2021, 09:17 Christoph Dahlen <notifications@github.com> ha
scritto:

**Reply by m0wlheld** [reply]
Copy of database and configuration should be sufficient, if you use the same coordinator device. 

But don’t blame me ...
